person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- [ ] Great job working on this milestone. As per project requirements,` id `should only be a getter.
- `Getters for @id, @name, and @age.`
- `Setters for @name and @age.`